bin/internal/update_dart_sdk.ps1

```diff
@@ -22,13 +22,15 @@ $dartSdkLicense = "$cachePath\LICENSE.dart_sdk_archive.md"
 $engineStamp = "$cachePath\engine-dart-sdk.stamp"
 $engineRealm = (Get-Content "$flutterRoot\bin\internal\engine.realm")
 
-if (Test-Path "$flutterRoot\bin\internal\engine.version") {
-    $engineVersion = (Get-Content "$flutterRoot\bin\internal\engine.version")
+PowerShell
```

Comment:
what does this do?

Reply:
That was me typing the wrong thing into the wrong window!

Reply:
... which like, power shell is OK with? huh

Reply:
lol